op-node/rollup/superchain.go

```diff
@@ -74,14 +74,6 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		regolithTime = 1692156862
 	}
 
-	deltaTime := superChain.Config.DeltaTime
-	// OP Labs Sepolia devnet 0 activated delta at genesis, slightly earlier than
-	// Base Sepolia devnet 0 on the same superchain.
-	switch chainID {
-	case labsSepoliaDevnet0:
-		deltaTime = new(uint64)
-	}
```

Comment:
We should add a test to `op-node/chaincfg/chains_test.go` for this network. It should fail right now.

Reply:
See [988e48e](https://github.com/ethereum-optimism/optimism/pull/9642/commits/988e48e5aa4bab93209a3f9022d5ffc6f44e0b75) and [bc10117](https://github.com/ethereum-optimism/optimism/pull/9642/commits/bc10117b06db0fbbd460474880c51f93ffa354a3).

Notes: 
* To construct the expected config for this chain, I copied everything over from `superchain-registry`
* This is the only devnet chain in this test suite

Reply:
* it fails as expected at commit [bc10117](https://github.com/ethereum-optimism/optimism/pull/9642/commits/bc10117b06db0fbbd460474880c51f93ffa354a3)